tests/data/dataset_readers/wikitables/wikitables_dataset_reader_test.py

```diff
@@ -113,6 +194,7 @@ def test_reader_reads(self):
                 "<e,r> -> fb:row.row.avg_attendance",
                 "<e,r> -> fb:row.row.division",
                 "<e,r> -> fb:row.row.league",
+                "<e,r> -> fb:row.row.null",  # null row, representing an empty set
```

Comment:
https://github.com/allenai/allennlp/pull/639/files/b50689afe6bcd2fc2953b9e53ad61428f9f03708#diff-e6b5c1785059fb13cf2030a09d3daf2eR44.

I'll fix the comment, here and in the place I linked.

Reply:
Thanks for the pointer. I missed that part earlier. Can you please also explain why you're generating null cells and columns? I didn't get that yet.

Reply:
Because DPD / sempre produce them.  If I'm remembering right, it's for places where you're checking that something is not an empty set (imagine, "is there a team that won three games in a row?").